src/components/two-up-display.tsx

```diff
@@ -93,38 +93,63 @@ export class TwoUpDisplayComponent extends BaseComponent<IProps, IState> {
           {this.props.rightPanel}
         </div>
         {this.renderRightPanelFooter()}
+        {this.renderFloatButtons()}
       </div>
```

Comment:
Float buttons are get absolute positioning and float somewhere in the right panel.